contracts/token/ERC20/IERC20.sol

```diff
@@ -20,6 +20,15 @@ interface IERC20 {
   function transferFrom(address from, address to, uint256 value)
     external returns (bool);
 
+  function increaseAllowance(address spender, uint256 addedValue) 
+    external returns (bool);
+
+  function decreaseAllowance(address spender, uint256 subtractedValue) 
+    external returns (bool);
+
+  function setAllowance(uint256 value) 
```

Comment:
The tests are failing because this function is making `ERC20` abstract (since the method is not implemented). This should only be added to the mock contracts, not the standard interface.

Reply:
got it.